extension/js/common/api/email-provider/gmail/gmail-parser.ts

```diff
@@ -65,6 +65,13 @@ export namespace GmailRes {
     nextPageToken: string;
     resultSizeEstimate: number;
   };
+  export type GmailThreadIdQuery = {
+    messages: {
+      id: string;
+      threadId: string;
+    }[];
+    resultSizeEstimate: number;
+  };
```

Comment:
we have `GmailMsgList` type which is the same as this one, let's just use it

Reply:
Thanks - I'll use  the `GmailMsgList`.